generators/client/templates/angular/package.json.ejs

```diff
@@ -67,7 +67,7 @@ limitations under the License.
     "@angular-builders/custom-webpack": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
     "@angular-builders/jest": "VERSION_MANAGED_BY_CLIENT_ANGULAR#devDependencies#@angular-builders/custom-webpack",
     "@angular/service-worker": "VERSION_MANAGED_BY_CLIENT_ANGULAR#dependencies#@angular/common",
-    "@angular-eslint/eslint-plugin": "0.5.0-beta.5",
+    "@angular-eslint/eslint-plugin": "0.6.0-beta.0",
```

Comment:
Add it to the package.json so dependabot can make the updates automatically?

Reply:
OK, will do it shortly.